packages/mux-player-react/src/common/utils.ts

```diff
@@ -11,6 +22,7 @@ const ReactPropToAttrNameMap = {
   playsInline: 'playsinline',
   autoPlay: 'autoplay',
   playbackRate: 'playbackrate',
+  defaultMuted: 'muted',
```

Comment:
issue(blocking): This is inaccurate. `muted` needs to apply dynamically over the life of the react player instance (e.g. if someone wants to programmatically toggle `muted` via props in their react app. There is also no attribute called `defaultMuted`. The `muted` HTML attribute will map to the `defaultMuted` HTML element property. Curious why the change was made here.

Reply:
> confusing stuff.
💯